diff --git a/node_modules/@sentry/tracing/cjs/browser/web-vitals/lib/getVisibilityWatcher.js b/node_modules/@sentry/tracing/cjs/browser/web-vitals/lib/getVisibilityWatcher.js
index 445cb49..71f889e 100644
--- a/node_modules/@sentry/tracing/cjs/browser/web-vitals/lib/getVisibilityWatcher.js
+++ b/node_modules/@sentry/tracing/cjs/browser/web-vitals/lib/getVisibilityWatcher.js
@@ -24,7 +24,7 @@ let firstHiddenTime = -1;
 const initHiddenTime = () => {
   // If the document is hidden and not prerendering, assume it was always
   // hidden and the page was loaded in the background.
-  return types.WINDOW.document.visibilityState === 'hidden' && !types.WINDOW.document.prerendering ? 0 : Infinity;
+  return (types.WINDOW.document && types.WINDOW.document.visibilityState === 'hidden') && !types.WINDOW.document.prerendering ? 0 : Infinity;
 };
 
 const trackChanges = () => {
