diff --git a/node_modules/react-native-modal-dropdown/components/ModalDropdown.js b/node_modules/react-native-modal-dropdown/components/ModalDropdown.js
index c7e4b4f..e1a7676 100644
--- a/node_modules/react-native-modal-dropdown/components/ModalDropdown.js
+++ b/node_modules/react-native-modal-dropdown/components/ModalDropdown.js
@@ -430,7 +430,7 @@ export default class ModalDropdown extends Component {
     return (
       <FlatList
         {...dropdownListProps}
-        getItemLayout={(data, index) => { return {length: 33 + StyleSheet.hairlineWidth, index, offset: (33 + StyleSheet.hairlineWidth) * index} }}
+        getItemLayout={(data, index) => { return { length: 55 + StyleSheet.hairlineWidth, index, offset: (55 + StyleSheet.hairlineWidth) * index } }}
         data={options}
         ref={component => (this.flatList = component)}
         scrollEnabled={scrollEnabled}
